### PR TITLE
Configuration of iSCSI installation

### PR DESCRIPTION
**Pull request by dzedro**
tested locally without support_server, because of issue https://progress.opensuse.org/issues/9482#note-11
http://10.100.98.90/tests/734/modules/iscsi_configuration/steps/3

### CODE REVIEW

File: tests/installation/iscsi_configuration.pm [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```
Comment (aaannz) [inline tests/installation/iscsi_configuration.pm]: Why keep insisting this file existed 7 years ago?

File: tests/installation/partitioning_iscsi.pm [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```
Comment (aaannz) [inline tests/installation/partitioning_iscsi.pm]: Same here

File: tests/installation/iscsi_configuration.pm (hunk 1 above)
Comment (dzedro) [inline tests/installation/iscsi_configuration.pm]: hmm, better now ?

Comment (coolo) [inline tests/installation/iscsi_configuration.pm]: bernhard has no copyright on this

Comment (dzedro) [inline tests/installation/iscsi_configuration.pm]: Are you sure ? :P

Comment (dzedro) [inline tests/installation/iscsi_configuration.pm]: updated ^^

**Pull request merged**